extract_unique_sensors_01.py

```diff
@@ -0,0 +1,39 @@
+from urllib.request import urlopen
+
+url = ('http://monctl.devm.camlab.kat.ac.za/kat/doc/manuals/sensors/SensorList.txt')
+try:
+    sensors_data = urlopen(url)
+except Exception:
+    filename = ('/home/lebukani/Downloads/SensorList.txt')
+    with open(filename) as lines:
+        sensors_data = lines.readlines()
+print(sensors_data)
+    
+def extract_all_sensors(lines): 
+    """
+    Filters the sensors_data and returns only a list of sensors.
+    
+    """
```

Comment:
- One-line docstring should fit on one line with quotes

Reply:
Something like """ Filters the sensors_data and returns only a list of sensors.""" ?